routers/products-router.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express')
+var bodyParser = require('body-parser');
+const ProductsRouter = express.Router();
+const cors = require('cors')
+ProductsRouter.use(cors())
```

Comment:
You don't need to initialize cors in every router file. Just initialize it in the main index.js file and it will work